### PR TITLE
Transition image to use node 6 to match AWS Lambda's architecture

### PR DESCRIPTION
**Pull request by gilday**
Use Node 6 to match AWS Lambda architecture. Remove unused tool chain components sam-local and assume role

### CODE REVIEW

**Review by rszewczyk — APPROVED** [review]
:shipit:

**Comment by gilday** [comment]
@rszewczyk I need you to :shipit: - you're the sole maintainer of this repo

File: README.markdown [hunk 1]
```diff
@@ -1,3 +1,4 @@
-# serverless
+# node-lambda
```
Comment (gilday) [inline README.markdown:1]: @rszewczyk what are your thoughts on this name for the repo? `serverless` is misleading so I am searching for a more appropriate name

Comment (rszewczyk) [reply README.markdown:1]: what's misleading about it?

**Pull request merged**